### ISSUE TITLE
Feature request's

### ISSUE THREAD

**Issue by LexterS999** [hongwenjun/vps_setup#10]
Hi, thanks for your script. It's help me a lot. But I want to ask to add some options:
1 Let users to choose which one port they can use by WG
2 Let users to choose which one udp2raw mode to use. For example, I need to use ICMP cause faketcp and udp mode doesn't help me to bypass ISP traffic shapping. Manually there is no way to change that mode in script for now. So for me there is only one way within ICMP mode.

But all other things work smoothly and perfect. Thanks for your work.

**Comment by hongwenjun** [comment]
1.服务端可以使用 bash wgmtu 修改；如果使用套接，服务端和客户端需要相同。
2. 编辑  /etc/rc.local  ，可以手动灵活设置(或者下载设置脚本 ，预置灵活修改https://github.com/hongwenjun/WinKcp_Launcher/blob/master/wg_udp2raw.sh)

**Comment by LexterS999** [comment]
1 It's hard for those who don't understand such things. Like me. I don't understand all that stuff
2 What about option to make it in script? I'm talking about option to choose custom port and especially custom raw mode for udp2raw

Best option setup script from your side so it must allow users to choose custom port or udp2raw mode initially by choosing numbers

**Comment by hongwenjun** [comment]
![](https://i.loli.net/2019/12/30/KOnQNRVDXcpqzAB.png)
设置脚本，可以修改Udp2raw 端口号的

**Comment by LexterS999** [comment]
Cool, thanks. But what about option to change udp2raw mode? Cause some ISP blocking tcp/udp so there is only one way - icmp

**Comment by hongwenjun** [comment]
> Cool, thanks. But what about option to change udp2raw mode? Cause some ISP blocking tcp/udp so there is only one way - icmp

udp2raw mode 还是手工好了，哪个容易太复杂，而不并不适合新手。
-icmp  当作最后的突防，还是留给熟悉的人使用

**Comment by LexterS999** [comment]
Yeah I know. That's why I asking you add option to choose which one udp2raw mode to use inside your script during bash wgmtu setup
